src/Tooltip/__tests__/Tooltip.test.js

```diff
@@ -16,7 +17,27 @@ describe('<Tooltip />', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
-  test('should render open a tooltip withouth a problem', () => {
+  test('should render without a problem with light appearance', () => {
+    const { getByTestId } = render(
+      <Tooltip
+        active
+        appearance="light"
+        title="Ventes nettes (ventes brutes moins les réductions et les annulations) plus les taxes
+        sur la période séléctionnée."
+      >
+        <button type="button">Show Tooltip</button>
+      </Tooltip>,
+    );
+    const tooltipNode = getByTestId('tooltip');
+
+    expect(tooltipNode).toHaveStyleRule({
```

Comment:
if I check the light css I see:
```js
    color: ${palette.darkBlue};
    background: ${palette.white};
    box-shadow: 0 0 0 1px ${palette.lightGrey}, 0 2px 16px 0 rgba(0, 0, 0, 0.1);
      &::before {
        background: ${palette.white};

```
is not what you test

Reply:
Thank you for that comment. I read the documentation again and I noticed my tests was totally wrong.